MissionHotpoint hotpoint_data
---
bool result
